integration-tests/src/test/java/com/datastax/oss/driver/mapper/NullSavingStrategyIT.java

```diff
@@ -125,11 +131,18 @@ public void should_do_not_throw_when_construct_dao_with_global_level_SET_TO_NULL
     ProductSimpleDaoGlobalLevelSetToNull productDaoGlobalLevelSetToNull(
         @DaoKeyspace CqlIdentifier keyspace);
 
+    @DaoFactory
+    ProductSimpleDaoSetToNullFromParentInterface productDaoSetToNullFromParentInterface(
+        @DaoKeyspace CqlIdentifier keyspace);
+
     @DaoFactory
     ProductSimpleDaoGlobalLevelDoNotSetOverrideSetToNull productDaoLocalOverride(
         @DaoKeyspace CqlIdentifier keyspace);
   }
 
+  @DefaultNullSavingStrategy(NullSavingStrategy.SET_TO_NULL)
```

Comment:
Will this work if the @`Mapper` will be annotated with `@DefaultNullSavingStrategy`?
If so maybe we should add a test for it.

Reply:
I don't believe so, at least I didn't intend for it to work that way.  We'd have to make `Mapper` annotations accessible for the Dao code, which is probably doable, but I think `Dao`-level and Daos supporting inheriting annotations may be powerful enough.